CHANGELOG.md

```diff
@@ -1,7 +1,168 @@
-# Rerun changelog
+6# Rerun changelog
 
 ## [Unreleased](https://github.com/rerun-io/rerun/compare/latest...HEAD)
 
+## [0.17.0](https://github.com/rerun-io/rerun/compare/0.16.1...0.17.0) - More Blueprint features and better notebooks - 2024-07-08
+
```

Comment:
We should write a migration guide and link to it here.

For one, it should cover how to know set the default point/line size, since that has changed. I'm sure there are other stuff too.